tests/unit/metadata/test_single_table.py

```diff
@@ -34,6 +66,514 @@ def test___init__(self):
             'SCHEMA_VERSION': 'SINGLE_TABLE_V1'
         }
 
+    def test__validate_numerical(self):
+        """Test the ``_validate_numerical`` method.
+
+        Setup:
+            - instance of ``SingleTableMetadata``
+            - list of accepted representations.
+
+        Input:
+            - Column name.
+            - sdtype numerical
+            - representation
+
+        Side Effects:
+            - Passes with the correct ``representation``
+            - ``ValueError`` is raised stating that the ``representation`` is wrong.
+        """
+        # Setup
+        instance = SingleTableMetadata()
+        representations = [
```

Comment:
You could parameterize the unit test with the list of representations. That way if one fails, it's easy to tell which representation failed.